viewer/src/datamodels/pointcloud/PotreeGroupWrapper.ts

```diff
@@ -46,7 +52,12 @@ export class PotreeGroupWrapper extends THREE.Object3D {
     }
   }
 
+  requestRedraw() {
+    this.redrawRequested = true;
+  }
```

Comment:
Can we omit `PotreeGroupWrapper` implementing `Redrawable` and rather handle this internally? I'm thinking we can e.g. have a `needsRedraw` in `PotreeNodeWrapper` which is set to true when its properties change and reset that field when `PotreeGroupWrapper.resetNeedsRedraw()` is invoked.